charts/pravega/templates/pravega.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
 {{ include "pravega.commonLabels" . | indent 4 }}
 spec:
-  {{- if .Values.tls }}
+ {{- if .Values.tls }}
```

Comment:
Any specific reason for changing the indentation here?

Reply:
No, I have changed it back.